modules/test/playwright/tests/osb-faro-web/acquisition.spec.ts

```diff
@@ -0,0 +1,474 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page, expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import getRandomString from '../../utils/getRandomString';
+import {syncAnalyticsCloud} from '../analytics-settings-web/utils/analyticsSettings';
+import {navigateToACSitesPageViaURL} from './utils/navigation';
+import {CardSelector} from './utils/selectors';
+import {closeSessions} from './utils/sessions';
+import {changeTimeFilter} from './utils/time-filter';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	loginAnalyticsCloudTest(),
+	loginTest()
+);
+
+async function sendEventByURL(page: Page, queryParams: string) {
```

Comment:
Hey @fmarcio, I think the name would be better if it were: `navigateToDXPByChannelViaURL` (or something related)

I think this way it is easier to understand, since we are not sending an event via the URL, we are generating the channel by accessing the DXP via the URL

Reply:
@ViniLopes20 done!